server/api/post.ts

```diff
@@ -0,0 +1,51 @@
+import { db } from "../database";
+import { isPost } from "../../server/types/DatabaseTypes";
+import { Request, Response } from "express";
+
+let firstPost: boolean = true;
+let postID: number;
+
+/**
+ * Creates a post, given valid Post information.
+ * @param {Request} req Requires a valid Post in the JSON body.
+ * @param {Response} res Responds with the post ID in the text field if successful, otherwise returns status code 500.
+ */
+function createPost(req: Request, res: Response) {
+    if (isPost(req.body)) {
+        db.run(`INSERT INTO Posts VALUES (?, ?, ?, ?, ?)`,
+            [postID, req.body.Username, req.body.Content, req.body.Picture, req.body.Timestamp],
+            function (err) {
+                if (err) {
+                    console.log("[SQL] Error!");
+                    res.status(500).send("Database error!");
```

Comment:
Specify the error by just outputting it. Also, you should be `return`ing after the send.